clients/pkg/promtail/positions/positions_test.go

```diff
@@ -40,7 +40,7 @@ func TestReadPositionsOK(t *testing.T) {
 	}()
 
 	yaml := []byte(`positions:
-  /tmp/random.log: "17623"
+  /log/path/random.log: "17623"
```

Comment:
hmm what about `./log`? my fear is that by using /log this might fail in some environments

Reply:
Doesn't seem like this path will actually be used.